tests/framework/microvm.py

```diff
@@ -552,6 +576,48 @@ def start(self):
         response = self.actions.put(action_type='InstanceStart')
         assert self._api_session.is_status_no_content(response.status_code)
 
+    def disable_netns(self):
+        """Disable network namespaces functionality, to use with cloning."""
+        # Cloning and network namespaces don't work well together yet.
```

Comment:
Could you add a little more detail about this (especially the `yet`part)?

Reply:
There is no support yet 😛 

This is a very basic test. @sandreim is working on adding a capable testing framework that can handle cloning.

Reply:
Got it, from from the comment it was not clear whether we were lacking in functionality or testing framework.